CHANGELOG.md

```diff
@@ -33,5 +38,6 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 
 [Unreleased]: https://github.com/cozy/cozy-client-js/compare/v0.1.4...HEAD
```

Comment:
can you update this like `https://github.com/cozy/cozy-client-js/compare/v0.1.5...HEAD`